server/services/commands/utils.js

```diff
@@ -0,0 +1,14 @@
+const config = require('../../config');
+const axios = require('axios');
+
+const sendCommand = async (req, res) => {
+  try{
+    const response = await axios.get(config.SERVERHOSTNAME, { params: req.query });
+    return response.data;
+  } catch (error) {
+    console.log("Error while calling the API : ", error);
+    return res.end({"success": false});
+  }
+}
+
+module.exports = { sendCommand }
```

Comment:
add new line at end of the file for all files, where github mark red circle. Or use `eslint --fix filepath.js`

Reply:
Fixed eslint errors.